docs/io/unified.rst

```diff
@@ -178,32 +178,74 @@ indicates which support write functionality.
 FITS
 ^^^^
 
-Reading/writing from/to `FITS <http://fits.gsfc.nasa.gov/>`_
-files is supported with ``format='fits'``. In most cases, existing FITS
-files should be automatically identified as such based on the header of the
-file, but if not, or if writing to disk, then the format should be explicitly
-specified.
+Reading and writing tables in `FITS <http://fits.gsfc.nasa.gov/>`_ format is
+supported with ``format='fits'``. In most cases, existing FITS files should be
+automatically identified as such based on the header of the file, but if not,
+or if writing to disk, then the format should be explicitly specified.
+
+Reading
+""""""""
 
 If a FITS table file contains only a single table, then it can be read in
 with::
 
+    >>> from astropy.table import Table
     >>> t = Table.read('data.fits')
 
-If more than one table is present in the file, the first table found will be
+If more than one table is present in the file, you can select the HDU
+as follows::
+
+    >>> t = Table.read('data.fits', hdu=3)
+
+In this case if the ``hdu`` argument is omitted then the first table found will be
 read in and a warning will be emitted::
 
     >>> t = Table.read('data.fits')
     WARNING: hdu= was not specified but multiple tables are present, reading in first available table (hdu=1) [astropy.io.fits.connect]
 
-To write to a new file::
+Writing
+""""""""
+
+To write a table ``t`` to a new file::
 
     >>> t.write('new_table.fits')
 
-At this time, the ``meta`` attribute of the
-:class:`~astropy.table.Table` class is simply an ordered
-dictionary and does not fully represent the structure of a FITS
-header (for example, keyword comments are dropped). This is likely
-to change in a future release.
+If the file already exists and you want to overwrite it, then set the
+``overwrite`` keyword::
+
+    >>> t.write('existing_table.fits', overwrite=True)
+
+At this time there is no support for appending an HDU to an existing file or
+writing multi-HDU files.
```

Comment:
Maybe amend this line to mention that this *is* possible, but only using the lower-level `io.fits` interface.
Though I think there are some things I can do to make this easier in the near term, such as allow a `BinTableHDU` to be created from a `Table`...